docs/docs/providers/keycloak.md

```diff
@@ -5,7 +5,11 @@ title: Keycloak
 
 ## Documentation
 
-https://www.keycloak.org/docs/latest/server_admin/#_oidc_clients
+[Official Keycloak Documentation](https://www.keycloak.org/docs/latest/server_admin/#_oidc_clients)
+
+For more details on creating an openid-connect client in Keycloak with "confidential" as the "Access Type", you can refer to the following resources:
+- [Keycloak Confidential Client Documentation](https://wjw465150.gitbooks.io/keycloak-documentation/content/server_admin/topics/clients/oidc/confidential.html)
```

Comment:
can you link an official doc link instead? this link doesn't look very legit IMO - moreover, it's 401 for me.